controllers/gcpcluster.go

```diff
@@ -25,13 +23,19 @@ type GCPClusterClient interface {
 	RemoveFinalizer(context.Context, *capg.GCPCluster, string) error
 }
 
+//counterfeiter:generate . FirewallsClient
+type FirewallsClient interface {
```

Comment:
Should this not live in the `pkg/firewall` module?

Reply:
I followed [go recommendations](https://github.com/golang/go/wiki/CodeReviewComments#interfaces)

> Go interfaces generally belong in the package that uses values of the interface type, not the package that implements those values.